Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -181,4 +181,31 @@ extension File {
         }
         return violatingRanges
     }
+
+    internal func numberOfCommentOnlyLines(startLine: Int, endLine: Int) -> Int {
```

Comment:
Right, that line could use a short comment explaining that, and these functions should be renamed accordingly.